Tests/js/views/fields/assets/ez-richtext-editview-tests.js

```diff
@@ -641,26 +610,6 @@ YUI.add('ez-richtext-editview-tests', function (Y) {
 
     });
 
-    destructorTest = new Y.Test.Case({
-        name: "eZ RichText View destructor test",
```

Comment:
you remove the test but the tested method is still there! See https://github.com/ezsystems/PlatformUIBundle/pull/440/files#r45455211